config-inject/package.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name": "@pleo-io/spa-config-inject",
+    "version": "0.0.1",
+    "license": "UNLICENSED",
+    "author": "Pleo",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/pleo-io/pleo-spa-cicd.git",
+        "directory": "config-inject"
+    },
+    "dependencies": {
+        "@commander-js/extra-typings": "^10.0.2",
+        "@types/jsesc": "^3.0.1",
+        "commander": "10.0.0",
+        "jsesc": "^3.0.2",
+        "lodash.merge": "^4.6.2",
+        "zod": "^3.21.4",
```

Comment:
I'm considering if zod should just be a peerDependency, to avoid having too many different zod version. Although, as this is not used runtime it isn't that important.

Reply:
I removed a bunch of dependencies, but I think zod needs to stay. If we use npx without installing first, it needs to be there, right?

Reply:
Ah yeah, that makes sense. But keeping the version pretty loose as you did makes sense then. 👍